Userland/Libraries/LibCore/Object.cpp

```diff
@@ -164,7 +164,12 @@ void Object::dump_tree(int indent)
 
 void Object::deferred_invoke(Function<void(Object&)> invokee)
 {
-    Core::EventLoop::current().post_event(*this, make<Core::DeferredInvocationEvent>(move(invokee)));
+    deferred_invoke([invokee { move(invokee) }, this] { invokee(*this); });
+}
+
+void Object::deferred_invoke(Function<void()> invokee)
+{
+    Core::deferred_invoke([invokee { move(invokee) }, strong_this = NonnullRefPtr(*this)] { invokee(); });
```

Comment:
Let's pick one and go with it, `invokee = move(invokee)` or `strong_this { NonnullRefPtr(*this) }`.